admin-dev/themes/new-theme/js/components/components-map.ts

```diff
@@ -29,16 +29,21 @@ export default {
     scrollbar: '.js-multistore-scrollbar',
   },
   multistoreHeader: {
+    header: '.header-multishop',
```

Comment:
```suggestion
    header: '.js-header-multishop',
```

Prefix every js-classes so we don't use these to style and we can remove/refacto them without thinking